kolibri/content/utils/import_export_content.py

```diff
@@ -61,3 +77,40 @@ def get_num_coach_contents(contentnode, filter_available=True):
             .count()
     else:
         return 1 if contentnode.coach_content else 0
+
+
+def retry_import(command, e, name, f=None, overall_progress_update=None):
```

Comment:
This needs a docstring! :)

Reply:
Yes thanks! I will add a docstring there